src/scss/components/_buttons.scss

```diff
@@ -5,8 +5,7 @@ $button-stroke-big: inset 0 0 0 units($theme-button-stroke-width * 2);
 
   &:not([disabled]):focus,
   &:not([disabled]).usa-focus {
-    @include disable-default-focus-styles;
-    box-shadow: roundable-focus-outline-box-shadows();
+    outline-offset: units($theme-focus-offset);
```

Comment:
We still need some overrides, since it appears that USWDS doesn't always use the configured theme variables.

See: https://github.com/uswds/uswds/issues/4543